Source/CarthageKit/Xcode.swift

```diff
@@ -1025,12 +1042,109 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 		}
 }
 
+
+public func getSecuritySigningIdentities() -> SignalProducer<String, CarthageError> {
+	let securityTask = TaskDescription(launchPath: "/usr/bin/security", arguments: ["find-identity", "-v", "-p", "codesigning"])
+	
+	return launchTask(securityTask)
+		|> ignoreTaskData
+		|> mapError { CarthageError.TaskError($0) }
+		|> map { (data: NSData) -> String in
+			return NSString(data: data, encoding: NSStringEncoding(NSUTF8StringEncoding))! as String
+		}
+		|> flatMap(.Merge) { (string: String) -> SignalProducer<String, CarthageError> in
+			return string.linesProducer |> promoteErrors(CarthageError.self)
+		}
+}
+
+public struct CodeSigningIdentity {
```

Comment:
This type seems like overkill because AFAICT we only every use the UUID. Can we just return that instead of capturing all the properties?